packages/wallet/frontend/src/lib/api/paymentPointer.ts

```diff
@@ -135,6 +142,7 @@ interface PaymentPointerService {
   ) => Promise<ListTransactionsResponse>
   generateKey: (args: GenerateKeyArgs) => Promise<GenerateKeyResponse>
   revokeKey: (args: RevokeKeyArgs) => Promise<RevokeKeyResponse>
+  assetCode: (url: string, cookies?: string) => Promise<AssetCodeResponse>
```

Comment:
Can not think of a proper name at the moment, but this should be changed.
```suggestion
  getExternal: (url: string, cookies?: string) => Promise<AssetCodeResponse>
```